### ISSUE TITLE
[ignition-gazebo] unable to initialize renderer

### ISSUE THREAD

**Issue by acxz** [acxz/gazebo-arch#2]
`gz sim shapes.sdf`:

```
[GUI] [Err] [RenderEngineManager.cc:517] Failed to load plugin [ogre2] : couldn't find shared library.
[GUI] [Err] [RenderEngineManager.cc:517] Failed to load plugin [ogre2] : couldn't find shared library.
```

**Comment by acxz** [comment]
upstream PR: https://github.com/gazebosim/gz-rendering/pull/553

**Comment by acxz** [comment]
Closing this issue as it can't be reproduced anymore

**Comment by acxz** [comment]
Keeping this issue open until `ogre-next` is added as a dep.

**Comment by goekce** [comment]
My PR #53 could be related. I was not aware of this current issue.

**Comment by acxz** [comment]
Closed with 3a3dad2
